assets/sass/components/_badges-tags.scss

```diff
@@ -1,8 +1,7 @@
-a.govuk-link {
-  font-weight: 400;
-}
-
 .govuk-table {
+  a.govuk-link {
+    font-weight: 400;
```

Comment:
Hey, in your other PR I saw you put: `font-weight: normal; ` - I didn't know it was a thing, but I think it's nicer than using the magic number of `400` - do you think you'd adding it here too? It's also nice to have consistency across the styles.

Reply:
sure, but it's truly not a big deal